tests/unit/conftest.py

```diff
@@ -0,0 +1,99 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2022 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import tempfile
+from pathlib import Path
+
+import pytest
+from craft_cli import messages
+
+
+class RecordingEmitter:
```

Comment:
I assume this is copied over? Maybe add a TODO to use a common package?

Reply:
This will be removed once testing fixtures are provided by craft-cli itself, or by pytest-craft-cli.